libs/deal-ticket/src/components/market-info/info-market.tsx

```diff
@@ -128,28 +131,34 @@ export const Info = ({ market }: InfoProps) => {
     },
     ...(market.accounts || [])
       .filter((a) => a.type === AccountType.ACCOUNT_TYPE_INSURANCE)
-      .map((a, i) => ({
+      .map((a) => ({
         title: t(`Insurance pool`),
         content: (
           <MarketInfoTable
             data={{
-              balance: `${a.balance}
-           ${market.tradableInstrument.instrument.product?.settlementAsset.symbol}`,
+              balance: a.balance,
             }}
+            assetSymbol={assetSymbol}
+            decimalPlaces={
+              market.tradableInstrument.instrument.product.settlementAsset
+                .decimals
+            }
           />
         ),
       })),
   ];
-
-  const keyDetails = pick(
-    market,
-    'name',
-    'decimalPlaces',
-    'positionDecimalPlaces',
-    'tradingMode',
-    'state',
-    'id' as 'marketId'
-  );
+  const { VEGA_EXPLORER_URL } = useEnvironment();
+  const keyDetails = {
+    ...pick(
+      market,
+      'name',
+      'decimalPlaces',
+      'positionDecimalPlaces',
+      'tradingMode',
+      'id' as 'marketId'
```

Comment:
question (blocking): Why the typecast, doesnt look like it does anything?

Reply:
it's not a typecast, it changes the key. Instead of displaying 'ID' in this section, it will display 'Market ID'. 
I followed this approach because I was trying to not miss any field from the market info that the API was giving me, maybe omit some controlled things like `__typename`, but also add new ones. this is why I was checking primitives and formatting keys. 
it makes it **super** easy to update the market info section. 
literally, one line of code is needed to control the fields. 
this seemed necessary when I was working on the market info section and I was going back and forth to update things. the list was getting super large, hundreds of lines of code, formatting seemed to be spread around, and it was repetitive. 
I am happy to discuss how to improve this component. this is why we were not following MarketInfoQuery types fully, .

Reply:
It is a typecast. All its doing here is typing the last argument you pass to `pick` as the literal string 'marketId' as opposed to it just being inferred as `string`. Its safe to remove here because the label text actually comes from the `marketID` key passed into the MarketInfoTable.

Reply:
Overall I think that rather than trying to ensure all the properties on the object are displayed we should more explicitly declare what fields we are showing. This should improve the type safety and simplify things quite a lot as its quite hard to follow all the data and additional props for formatting.

Reply:
That said, I think lets merge this and revisit it